<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WRamMapEditor</class>
 <widget class="QWidget" name="WRamMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WRam Map Editor</string>
  </property>
  <widget class="QTableView" name="mapEntryTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>291</width>
     <height>551</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>search :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serarchLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="WRamEntryEditor" name="wramEntryEditor" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="WRamEntryDisplay" name="wramEntryDisplay" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>681</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WRamEntryEditor</class>
   <extends>QWidget</extends>
   <header>wramentryeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WRamEntryDisplay</class>
   <extends>QWidget</extends>
   <header>wramentrydisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
